src/screens/EncounterDataScreen.tsx

```diff
@@ -0,0 +1,214 @@
+import {
+  launchImageLibraryAsync,
+  MediaTypeOptions,
+  PermissionStatus,
+  requestMediaLibraryPermissionsAsync
+} from 'expo-image-picker';
+import { noop } from 'lodash';
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  Alert,
+  Pressable,
+  ScrollView,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import { normalize } from 'react-native-elements';
+
+import {
+  Button,
+  DateTimePicker,
+  EncounterList,
+  ImageWithBadge,
+  Label,
+  LoadingSpinner,
+  SafeAreaViewFlex,
+  SectionHeader,
+  Wrapper,
+  WrapperRow,
+  WrapperWithOrientation
+} from '../components';
+import { colors, Icon, texts } from '../config';
+import { momentFormat } from '../helpers';
+import { useEncounterUser } from '../hooks';
+
+// TODO: accesibility labels
+export const EncounterDataScreen = () => {
+  const [isDatePickerVisible, setIsDatePickerVisible] = useState(false);
+  const [imageUri, setImageUri] = useState<string>();
+  const [givenName, setGivenName] = useState<string>();
+  const [familyName, setFamilyName] = useState<string>();
+  const [birthDate, setBirthDate] = useState<Date>();
+  const [phone, setPhone] = useState<string>();
+
+  const userData = useEncounterUser();
+
+  // TODO: implement
+  const updateUserData = noop;
+
+  const selectImage = useCallback(async () => {
+    const { status } = await requestMediaLibraryPermissionsAsync();
+
+    if (status !== PermissionStatus.GRANTED) {
+      Alert.alert(texts.errors.image.title, texts.errors.image.body);
+      return;
+    }
+
+    const result = await launchImageLibraryAsync({
+      mediaTypes: MediaTypeOptions.Images,
```

Comment:
an info somewhere or link to https://docs.expo.dev/versions/latest/sdk/imagepicker/#imagepickermediatypeoptions would have been nice.
so we do not want videos selectable but just images, what this option realizes.